apps/spkac.c

```diff
@@ -67,7 +67,7 @@ int spkac_main(int argc, char **argv)
         switch (o) {
         case OPT_EOF:
         case OPT_ERR:
- opthelp:
+opthelp:
```

Comment:
There is great ambiguity in how labels of this kind are formatted. Single space as indentation is actually more common...

Reply:
No problem, I'll revert it.